src/OrbitGl/GpuTrack.cpp

```diff
@@ -132,19 +131,21 @@ void GpuTrack::Draw(Batcher& batcher, TextRenderer& text_renderer, uint64_t curr
     submission_track_->SetSize(size_[0], submission_track_->GetHeight());
   }
 
-  Track::Draw(batcher, text_renderer, current_mouse_time_ns, picking_mode, z_offset);
+  Track::Draw(batcher, text_renderer, draw_context);
 
   if (collapse_toggle_->IsCollapsed()) {
     return;
   }
-
+  const DrawContext& sub_track_draw_context = {
```

Comment:
I'm not sure if this is the best form to create a new DrawContext. Another option is used when we set the z_offset in TimeGraph.cpp (copy it first and modify the value after). I think that the TimeGraph's one could be better and for sure we should only use one of them. What do you think?

Reply:
Perfect, we agree there :)

Reply:
+1 for copying and then changing the copy